.teamcity/Gradle_Check/configurations/Gradleception.kt

```diff
@@ -19,7 +19,7 @@ class Gradleception(model: CIBuildModel, stage: Stage) : BaseGradleBuildType(mod
     val buildScanTagForType = buildScanTag("Gradleception")
     val defaultParameters = (gradleParameters() + listOf(buildScanTagForType)).joinToString(separator = " ")
 
-    applyDefaults(model, this, ":install", notQuick = true, extraParameters = "-Pgradle_installPath=dogfood-first $buildScanTagForType", extraSteps = {
+    applyTestDefaults(model, this, ":install", notQuick = true, extraParameters = "-Pgradle_installPath=dogfood-first $buildScanTagForType", extraSteps = {
```

Comment:
This would try to rerun the `:install` step and not the step actually running the tests. Let's not try to re-run `Gradleception` - it only runs unit tests, and those should not be flaky.

Reply:
The issue is not flakiness - the issue was, if the Gradleception's unit tests fail, current tagging process can't distinguish this failure and the real flaky test failure: if a build has only one `TC_FAILED_TESTS` problem, the tagging process would think it as flaky. So, in this case, tagging process would mark the build as successful and conceal real problems.

Reply:
Then we should fix our tagging, not add the re-run step: The re-run step will re-run `install`, since that is the task passed to `applyTestDefaults`.

I think our tagging should check if we have two steps, and only then apply the flaky test tagging.